core/src/test/scala/org/apache/spark/metrics/sink/GraphiteSinkSuite.scala

```diff
@@ -79,4 +79,42 @@ class GraphiteSinkSuite extends SparkFunSuite {
     assert(metricKeys.equals(filteredMetricKeys),
       "Should contain only metrics matches regex filter")
   }
+
+  test("GraphiteSink without host") {
```

Comment:
shall we have one big test suite for spark core errors? like `SparkCoreErrorSuite`? We have several of them in sql, such as `QueryCompilationErrorsSuite`